src/gt4py/stencil_object.py

```diff
@@ -240,12 +240,16 @@ def _call_run(self, field_args, parameter_args, domain, origin, exec_info=None):
             origin.setdefault(name, origin["_all_"] if "_all_" in origin else field.default_origin)
 
         # Domain
-        max_domain = Shape([sys.maxsize] * self.domain_info.ndims)
+        max_domain = Shape([np.iinfo(np.uintc).max] * self.domain_info.ndims)
```

Comment:
today I learned!

Reply:
although, in principle domain sized could be python integers, then it could actually be infinite